FAQ.md

```diff
@@ -30,6 +30,9 @@
   * [Which additional features on top of a GUI does BIT provide over a self-configured rsync backup? I saw that it saves the names for uids and gids, so I assume it can restore correctly even if the ids change. Great! :-) Are there additional benefits?](#which-additional-features-on-top-of-a-gui-does-bit-provide-over-a-self-configured-rsync-backup-i-saw-that-it-saves-the-names-for-uids-and-gids-so-i-assume-it-can-restore-correctly-even-if-the-ids-change-great---are-there-additional-benefits)
   * [How to move snapshots to a new hard-drive?](#how-to-move-snapshots-to-a-new-hard-drive)
   * [How to move large directory in source without duplicating backup?](#how-to-move-large-directory-in-source-without-duplicating-backup)
+- [Testing & Building](#testing--building)
+  * [SSH related tests are skipped](#ssh-related-tests-are-skipped)
+  * [Setup SSH Server to run unit tests](#setup-ssh-server-to-run-unit-tests)
```

Comment:
great, I was trying hard to find our ssh server setup instructions last weekend :-)